article-analyzer/src/main/java/co/inventorsoft/academy/spring/articleanalyzer/service/SetToJsonWriter.java

```diff
@@ -0,0 +1,31 @@
+package co.inventorsoft.academy.spring.articleanalyzer.service;
+
+import com.google.gson.Gson;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Set;
+
+@Service("setToJsonWriter")
```

Comment:
remove all of these custom bean names across other classes, spring does create bean with exactly the name is set here